accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -1206,3 +1283,13 @@ understand these changes: for example, by providing a debugging option in which
 instance checks or casts which would result in a different outcome if run in
 strong checking mode vs weak checking mode are flagged for the developer by
 logging a warning or breaking to the debugger.
+
+### Automatic debug assertion insertion
```

Comment:
@johnniwinther @stefantsov This change is lower priority, but something that I'd like to do if we have time before the release.